pkg/apis/configs/brokerdelivery/br_delivery_defaults_test.go

```diff
@@ -116,7 +117,7 @@ func TestNewDefaultsConfigFromConfigMapWithError(t *testing.T) {
 		"empty data": {
 			config: &corev1.ConfigMap{
 				ObjectMeta: metav1.ObjectMeta{
-					Namespace: "cloud-run-events",
+					Namespace: system.Namespace(),
```

Comment:
This will evaluate to "knative-testing" instead of "events-system" because this file includes:

`import _ "knative.dev/pkg/system/testing"`

which initialize the value to  "knative-testing".

The test seem to be passing anyway because it doesn't care about the namespace value.

Reply:
As discussed, I removed the commit that made these changes. The use of `system.Namespace()` will be delegated to solving #1958.